tower-grpc-interop/src/client.rs

```diff
@@ -216,6 +216,302 @@ fn assert_success(result: Result<Vec<TestAssertion>, Box<Error>>)
     })
 }
 
+type TestClient = pb::client::TestService<tower_http::AddOrigin<
+    tower_h2::client::Connection<
+        tokio_core::net::TcpStream, tokio_core::reactor::Handle, tower_h2::BoxBody>>>;
+
+fn empty_unary_test(client: &mut TestClient, core: &mut tokio_core::reactor::Core)
```

Comment:
Thanks, breaking these out is much nicer to read IMHO.

Take it or leave it: if you defined `TestClient` as a newtype instead,
```rust
struct TestClient(
pb::client::TestService<tower_http::AddOrigin<
    tower_h2::client::Connection<
        tokio_core::net::TcpStream, tokio_core::reactor::Handle, tower_h2::BoxBody>>>
)
```
you could then define these functions as methods:
```rust
impl TestClient {
    fn empty_unary_test(&mut self, core: &mut tokio_core::reactor::Core) -> Result<Vec<TestAssertion>, Box<Error>> {
        // ...
    }

    fn large_unary_test(&mut self, core: &mut tokio_core::reactor::Core) -> Result<...> {
        // ...
    }

    fn cacheable_unary_test(&mut self, ...) -> Result<...> {
        // ...
    }
}
```
and so on.

Reply:
Done.